### ISSUE TITLE
[BUG] Request postData() always returns null 

### ISSUE THREAD

**Issue by karamfilovs** [microsoft/playwright#9648]
**Context:**
- Playwright Version: 1.15.2 (Java binding)
- Operating System: Linux, Windows, MacOs
- Browser: Chrome, Edge
- The issue is reproducible for js version of the library as well.

**Code Snippet**

Listening for request with specific url pattern stopped returning postData.

```javascript
Log:
2021:10:20:14:31:11.742  INFO  MoviePage - Request url: */beat
2021:10:20:14:31:11.745  INFO  MoviePage - Headers:{sec-ch-ua="Chromium";v="94", "Microsoft Edge";v="94", ";Not A Brand";v="99", referer=https://*/, sec-ch-ua-mobile=?0, user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36 Edg/94.0.992.50, sec-ch-ua-platform="Windows", content-type=application/json}
2021:10:20:14:31:11.747  INFO  MoviePage - Data:null
2021:10:20:14:31:11.748  INFO  MoviePage - Method:POST
```

**Describe the bug**
Launch browser
Listen for request by specific url pattern
Print request details url, postData, headers

The above test used to work for few months without any issues. I was able to retrive postData and analyze it

The request postData is visible in the browser network tab so its definately not null. It looks like a problem in the communication between browser and playwright library.
![beat](https://user-images.githubusercontent.com/4948071/138089142-bbc32711-fdc8-48c5-8001-70bdfceee72a.png)
.

**Comment by mxschmitt** [comment]
Merging into #6479

**Comment by karamfilovs** [comment]
Just bumped to 1.17 and the issue is still reproducible. Is the fix included in v1.17?

**Comment by mxschmitt** [comment]
It is not fixed yet, the issue was just merged into #6479.

**Comment by karamfilovs** [comment]
Thanks I was probably mislead by the tag v1.17 above.
